peps/pep-0739.rst

```diff
@@ -0,0 +1,131 @@
+PEP: 739
+Title: Static description file for Python installations
+Author: Filipe Laíns <lains@riseup.net>
+Status: Draft
+Type: Informational
```

Comment:
This should be changed, see https://peps.python.org/pep-0001/#pep-types
```suggestion
Type: Standards Track
```

And is it Topic: Packaging?

Reply:
> And is it Topic: Packaging?

Yeah, I think that makes sense.

Reply:
Then please also add Paul as the PEP-Delegate, like in:

https://github.com/python/peps/blob/main/peps/pep-0735.rst?plain=1